utils/freifunk-berlin-autoupdate/Makefile

```diff
@@ -0,0 +1,66 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=freifunk-berlin-autoupdate
+PKG_VERSION:=0.9.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/Akira25/autoupdate.git
+PKG_SOURCE_DATE:=2019-05-20
+PKG_SOURCE_VERSION:=89997c4ba662eac86a7f8359b25a3b196437ffa9
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/freifunk-berlin-autoupdate/default
+  SECTION:=freifunk-berlin
+  CATEGORY:=freifunk-berlin
+  URL:=https://github.com/Akira25/autoupdate
+  PKGARCH:=all
+endef
+
+define Package/freifunk-berlin-autoupdate
+  $(call Package/freifunk-berlin-autoupdate/default)
+  TITLE:=A script trying to get the upgrade process of a freifunk-berlin router smooth and easy
+  DEPENDS:=+uci
+endef
+
+define Package/freifunk-berlin-autoupdate/description
+  autoupdate wants to get the upgrade process of a freifunk-berlin router via terminal smooth and easy.
+endef
+
+define Build/Prepare
+endef
+
+define Build/Configure
+endef
+
+define Build/Compile
+endef
+
+define Package/freifunk-berlin-autoupdate/install
+	$(INSTALL_DIR) $(1)/usr/bin
+        $(INSTALL_BIN) $(PKG_BUILD_DIR)/files/autoupdate $(1)/usr/bin/autoupdate
```

Comment:
from http://buildbot.berlin.freifunk.net/builders/ramips-mt7620/builds/1264/steps/shell/logs/stdio
> make[3]: [package/feeds/packages_berlin/freifunk-berlin-autoupdate/compile] Error 2 (ignored)
> package/Makefile:107: recipe for target 'package/feeds/packages_berlin/freifunk-berlin-autoupdate/compile' failed
> Makefile:66: *** missing separator (did you mean TAB instead of 8 spaces?).  Stop.
> time: package/feeds/packages_berlin/freifunk-berlin-autoupdate/compile#0.02#0.00#0.09
> make[3]: *** [package/feeds/packages_berlin/freifunk-berlin-autoupdate/compile] Error 2

Reply:
I replaced the spaces with TABs. The new version is here:
https://github.com/Akira25/firmware-packages/commit/4d0bc5abb412a645a688197d6da2c332daa9336a
Hopefully it will work that time.